Formula/libtrng.rb

```diff
@@ -0,0 +1,34 @@
+class Libtrng < Formula
+  desc "Tina's Random Number Generator Library"
+  homepage "https://www.numbercrunch.de/trng/"
+  url "https://www.numbercrunch.de/trng/trng-4.19.tar.gz"
+  sha256 "36b49961c631ae01d770ff481796c8b280e18c6b6e6b5ca00f2b868533b0492e"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <trng/yarn2.hpp>
+      #include <trng/normal_dist.hpp>
+      int main()
+      {
+        trng::yarn2 R;
+        trng::normal_dist<> normal(6.0, 2.0);
+        (void)normal(R);
+        return 0;
+      }
+    EOS
+    flags = %W[
```

Comment:
Do not use an array, simply inline the flags:

`system ENV.cxx, "test.cpp", "-o", "test", "-I#{include}", "-L#{lib}", "-ltrng4"`

(and note the added `-L#{lib}`)